Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -1049,9 +1059,14 @@ GRefPtr<GstElement> GStreamerMediaEndpoint::requestAuxiliarySender(GstWebRTCDTLS
         return nullptr;
     }
 
-    auto estimator = adoptGRef(makeGStreamerElement("rtpgccbwe", nullptr));
+    // Don't use makeGStreamerElement() here because it would be called mutiple times and emit an
+    // error every single time if the element is not found.
+    auto estimator = adoptGRef(gst_element_factory_make("rtpgccbwe", nullptr));
```

Comment:
You can't adopt a floating reference, this will crash in debug mode, please change this before landing.

Reply:
Ah funny, this was an issue in previous code too.